internal/metrics/metrics.go

```diff
@@ -187,6 +227,172 @@ func GetScalingSignal(pdb *policyv1.PodDisruptionBudget) string {
 	return PDBBlockedSignal
 }
 
+// GetScalingOpportunityLabels gets age pattern and effectiveness labels for scaling opportunities
+func GetScalingOpportunityLabels(ctx context.Context, c client.Client, pdb *policyv1.PodDisruptionBudget) (string, string) {
+	pattern, err := AnalyzePodAgeFailurePattern(ctx, c, pdb)
+	if err != nil {
+		return "unknown", "unknown"
+	}
+	
+	isHelpful := PredictScalingEffectiveness(pattern)
+	helpfulLabel := UnlikelyHelpfulStr
+	if isHelpful {
```

Comment:
nit is isHelpful uncessary variable also PredictScalingEffectiveness can return a string instead a bool if this is the only place used.